examples/all-clusters-app/all-clusters-common/gen/endpoint_config.h

```diff
@@ -391,232 +418,351 @@
 // Array of networks supported on each endpoint
 #define FIXED_NETWORKS                                                                                                             \
     {                                                                                                                              \
-        0, 0                                                                                                                       \
+        1, 1                                                                                                                       \
     }
 
-#define EMBER_AF_GENERATED_PLUGIN_INIT_FUNCTION_DECLARATIONS                                                                       \
-    void emberAfPluginReportingInitCallback(void);                                                                                 \
-    void emberAfPluginTemperatureMeasurementServerInitCallback(void);                                                              \
-    void emberAfPluginBarrierControlServerInitCallback(void);                                                                      \
-    void emberAfPluginDoorLockServerInitCallback(void);
-
-#define EMBER_AF_GENERATED_PLUGIN_INIT_FUNCTION_CALLS                                                                              \
-    emberAfPluginReportingInitCallback();                                                                                          \
-    emberAfPluginTemperatureMeasurementServerInitCallback();                                                                       \
-    emberAfPluginBarrierControlServerInitCallback();                                                                               \
-    emberAfPluginDoorLockServerInitCallback();
-
-#define EMBER_AF_GENERATED_PLUGIN_STACK_STATUS_FUNCTION_DECLARATIONS                                                               \
-    void emberAfPluginReportingStackStatusCallback(EmberStatus status);                                                            \
-    void emberAfPluginTemperatureMeasurementServerStackStatusCallback(EmberStatus status);                                         \
-    void emberAfPluginIasZoneServerStackStatusCallback(EmberStatus status);
-
-#define EMBER_AF_GENERATED_PLUGIN_STACK_STATUS_FUNCTION_CALLS                                                                      \
-    emberAfPluginReportingStackStatusCallback(status);                                                                             \
-    emberAfPluginTemperatureMeasurementServerStackStatusCallback(status);                                                          \
-    emberAfPluginIasZoneServerStackStatusCallback(status);
-
-// Generated data for the command discovery
+// Array of EmberAfCommandMetadata structs.
+#define ZAP_COMMAND_MASK(mask) COMMAND_MASK_##mask
+#define EMBER_AF_GENERATED_COMMAND_COUNT (165)
```

Comment:
Do you know why we are going from `161`to `165` ?

Reply:
From memory it is because the commands between endpoint are not regrouped anymore with zap. Which leads to the duplication of commands for the second endpoint with the On/Off cluster thus increasing the Command Count.